### PR TITLE
drop import of github.com/stretchr/testify

### PR DESCRIPTION
**Pull request by gy95**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
drop import of github.com/stretchr/testify, instead use go native.
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: edge/pkg/devicetwin/dtmanager/membership_test.go [hunk 1]
```diff
@@ -87,8 +93,9 @@ func TestDealMembershipDetailInvalidMsg(t *testing.T) {
 	}
 
 	err := dealMembershipDetail(dtc, "t", m)
-	assert.Error(t, err)
-	assert.Equal(t, errors.New("assertion failed"), err)
+	if !reflect.DeepEqual(err, errors.New("assertion failed")) {
```
Comment (fisherxu) [inline edge/pkg/devicetwin/dtmanager/membership_test.go:96]:
```suggestion
	if err.Error()!="assertion failed" {
```
How about this?

Comment (gy95) [reply edge/pkg/devicetwin/dtmanager/membership_test.go:96]: both are ok, but I think it's better to judge `error` itself, that's more accurate.

Comment (fisherxu) [reply edge/pkg/devicetwin/dtmanager/membership_test.go:96]: /approve

**Review by fisherxu — COMMENTED** [review]
/lgtm

**Pull request merged**